deploy/helm-chart/kube-mail/templates/rbac.yaml

```diff
@@ -26,11 +28,13 @@ kind: ClusterRole
 apiVersion: rbac.authorization.k8s.io/v1
 metadata:
   name: {{ template "chart.fullname" . }}-podwatcher
-  labels:
-    app.kubernetes.io/name: {{ template "chart.name" . }}
-    helm.sh/chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
-    app.kubernetes.io/instance: {{ .Release.Name }}
-    app.kubernetes.io/managed-by: {{ .Release.Service }}
+  labels: {{ include "chart.labels" . | nindent 4 }}
+    {{- if .Values.commonLabels }}
+    {{- toYaml .Values.commonLabels | nindent 4}}
+    {{- end }}
```

Comment:
Since up until now I've only seen `chart.labels` and `.Values.commonLabels` being used together: Does it make sense to just include the `if .Values.commonLabels` snippet into the `chart.labels` macro? Or maybe to create a new macro that combines these two snippets?

Reply:
> if you want to avoid code duplication, i would implement
a new macro (chart.commonLabels), which essentially
calls chart.labels and also renders the commonLabels

Sounds good to me, actually. :+1: